target/dovecot/dovecot-oauth2.conf.ext

```diff
@@ -0,0 +1,3 @@
+client_id = verySecretId
+client_secret = verySecretSecret
+introspection_url = https://oauth2.example.com/userinfo/
```

Comment:
1 & 2 sorted, looking into the compose feature now. I didn't even know this was a thing.

Reply:
You know what, especially since it's now just a one line file... This might be a good shout.

I don't have the experience to make the call though, I'll leave it up to you. Let me know if you'd like me to set it up using compose instead.